templates/server.conf.j2

```diff
@@ -13,11 +13,19 @@ server-bridge nogw
 
 dev tap
 {% else %}
-server {{openvpn_tunnel_network}} {{openvpn_tunnel_netmask}} nopool
-ifconfig-pool {{openvpn_tunnel_dhcp_range_start}} {{openvpn_tunnel_dhcp_range_end}}
-    {%- if openvpn_tunnel_ipv6_network is defined %}
-
-server-ipv6 {{openvpn_tunnel_ipv6_network}}
+server {{ openvpn_tunnel_subnetv4 | ipaddr('network') }} {{ openvpn_tunnel_subnetv4 | ipaddr('netmask') }} nopool
+{% set ipv4_range_start = openvpn_tunnel_subnetv4 | ipaddr(openvpn_tunnel_dynamic_ipv4_range_start | ipaddr('int')) %}
```

Comment:
Maybe use `next_nth_usable` instead `ipaddr` to improve readability.

Reply:
But then I would need to subtract 1.

Reply:
I was able to use `nthhost` here. But not below for IPv6, because there we need to keep the `/64`. `ipaddr(some_number)` keeps the CIDR suffix whereas `nthhost(some_number)` doesn't.